data/api.ts

```diff
@@ -2,6 +2,7 @@ export async function fetchEndpoint(url: string, method: string, data?: any) {
   return fetch(url, {
     method,
     headers: {
+      ...((!data || data !== 0) && { 'Content-Length': '0' }),
```

Comment:
Looks like a small typo here, should be `&&` instead of `||`
`...(!data && data !== 0 && { 'Content-Length': '0' }),`